readme.md

```diff
@@ -1,9 +1,9 @@
-This repository contains the command line tool (`Octo.exe`) for [Octopus Deploy][1], an automated deployment server for professional .NET developers. You can use it to create and deploy releases, create and push packages, and manage environments.
+This repository contains the command line tool (`octo.exe`) for [Octopus Deploy][1], an automated deployment server for professional .NET developers. You can use it to create and deploy releases, create and push packages, and manage environments.
```

Comment:
This should get the rename treatment. `Octopus CLI` (or whatever it should be). Also on line 3 and 6.

While we're at it, get @robpearson's or @MJRichardson's input on rewording `an automated deployment server for professional .NET developers`.

Reply:
Okay, done. Negotiated a refreshed one-line description with @robpearson and I've also applied it to the OctopusDeploy repo's description and `readme` opening sentence.